src/main/java/com/modrinth/minotaur/dependencies/DependenciesConfiguation.java

```diff
@@ -0,0 +1,70 @@
+package com.modrinth.minotaur.dependencies;
+
+import javax.inject.Inject;
+
+import org.gradle.api.NamedDomainObjectContainer;
+import org.gradle.api.model.ObjectFactory;
+
+import com.modrinth.minotaur.dependencies.container.DependencyContainer;
+import com.modrinth.minotaur.dependencies.container.DependencyObjectContainer;
+
+/**
+ * the Nested Dependencies configuration
+ */
+public class DependenciesConfiguation  {

```

Comment:
Typo, should be `DependenciesConfiguration`

Reply:
Ouch, that's embarrassing. Yep will fix

Reply:
Since i had to change how this class was defined I opted for class name changes to, imo, better define the classes. Not ashamed to admit I'm not the best at all at well written Javadoc's so if any don't make sense let me know.